utility/utils.py

```diff
@@ -750,7 +749,63 @@ def download_openshift_installer(version=defaults.INSTALLER_VERSION):
             f"https://mirror.openshift.com/pub/openshift-v4/clients/ocp/"
             f"{version}/openshift-install-{os_type}-{version}.tar.gz"
         )
+        # Prepare BIN_DIR
+        try:
+            os.mkdir(defaults.BIN_DIR)
+        except FileExistsError:
+            pass
```

Comment:
I would at least write debug message to log about that bin directory exist instead of pass WDYT?

Reply:
Addressed in 0056ce6f3e59d0365c7ef238bd4b29cd3b1e36bc.